ftp/1.5/modules/ROOT/pages/ftp-connection.adoc

```diff
@@ -0,0 +1,36 @@
+= Connect to an FTP Server Example - Mule 4
+
+To configure the global element to connect to an FTP Server, follow these steps:
```

Comment:
"FTP server" is not a proper noun, a name that "belongs" to one thing or one company or one person. So the "server" part is lowercase (except in titles or, perhaps, as a UI label):
```suggestion
To configure the global element to connect to an FTP server, follow these steps:
```